containers/python/same_prediction_container.py

```diff
@@ -0,0 +1,66 @@
+from __future__ import print_function
```

Comment:
This is even more noop than the noop container -- I am using it for throughput benchmarking to minimze the amount of time actually generating predictions, letting container latency better approximate rpc overhead. This isn't particularly hard to re-implement when needed; it might just be muddying the codebase to have this here. Happy to remove if you think that's best.